### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Abdulazeez-Dolapo**
This pull request is a fix for issue #171 - Read and write event types from the database.

### CODE REVIEW

File: src/components/MeetingTypes/MeetingTypeCard.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  state = {
+    editing: false,
+    loading: false,
+  };
+
+  handleFormOpen = () => {
+    this.setState({ editing: !this.state.editing });
+  };
+
+  // handleFormSubmit = (formValues) => {
```
Comment (meganrm) [inline src/components/MeetingTypes/MeetingTypeCard.js:20]: delete?

Comment (Abdulazeez-Dolapo) [reply src/components/MeetingTypes/MeetingTypeCard.js]: I've deleted the commented code. I'm assuming that's what you mean.

Comment (meganrm) [reply src/components/MeetingTypes/MeetingTypeCard.js]: yeah

File: src/state/meeting-types/logic.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    return query
+      .once("value")
+      .then((snapshot) => {
+        if (snapshot.empty) {
+          console.log("No Meeting types found");
+          return;
+        }
+
+        snapshot.forEach((meetingType) => {
+          const meetingTypeData = meetingType.val();
```
Comment (meganrm) [inline src/state/meeting-types/logic.js:37]: I didn't have ids on my data when I uploaded it
```suggestion
          const meetingTypeData = meetingType.val();
          meetingTypeData.id = meetingType.key;
```

File: src/components/MeetingTypes/MeetingTypeCard.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      description,
+      icon: <Icon type="smile" style={{ color: "#108ee9" }} />,
+      duration: 3.0,
+    });
+  };
+
+  render() {
+    const { Text } = Typography;
+
+    const { name, description, icon, color, display } = this.props.meetingType;
```
Comment (meganrm) [inline src/components/MeetingTypes/MeetingTypeCard.js:42]:
```suggestion
        const { name, description, icon, color, display, text, email } = this.props.meetingType;
```

File: src/components/MeetingTypes/MeetingTypeCard.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            <p>
+              <Text strong>Description: </Text> {description}
+            </p>
+            <p>
+              <Text strong>Map Icon: </Text> {this.getNameFromIconFlags(icon)}
+            </p>
+
+            <div className="tag">
+              {display ? <Tag color="red">Include in table</Tag> : ""}
+            </div>
```
Comment (meganrm) [inline src/components/MeetingTypes/MeetingTypeCard.js:63]:
```suggestion
            <div className="tag">
              {email ? <Tag color="red">Include in email alerts</Tag> : ""}
            </div>
            <div className="tag">
              {text ? <Tag color="red">Include in text alerts</Tag> : ""}
            </div>
        </div>
```

File: src/components/MeetingTypes/MeetingTypeCard.js (hunk 3 above)
Comment (meganrm) [reply src/components/MeetingTypes/MeetingTypeCard.js:42]: just missing the tags for the other 2 checkboxes

File: src/components/MeetingTypes/EditMeetingTypeForm.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+                </Option>
+              ))}
+            </Select>
+          )}
+        </Form.Item>
+
+        <Form.Item label="Description" className="input">
+          {getFieldDecorator("description", {
+            initialValue: description,
+            rules: [{ required: true, message: "Please input description!" }],
```
Comment (meganrm) [inline src/components/MeetingTypes/EditMeetingTypeForm.js:91]: this shouldn't be required

File: src/state/meeting-types/logic.js (hunk 2 above)
Comment (Abdulazeez-Dolapo) [reply src/state/meeting-types/logic.js:37]: done

File: src/components/MeetingTypes/MeetingTypeCard.js (hunk 3 above)
Comment (Abdulazeez-Dolapo) [reply src/components/MeetingTypes/MeetingTypeCard.js]: done

File: src/components/MeetingTypes/MeetingTypeCard.js (hunk 4 above)
Comment (Abdulazeez-Dolapo) [reply src/components/MeetingTypes/MeetingTypeCard.js:71]: done

File: src/components/MeetingTypes/EditMeetingTypeForm.js (hunk 5 above)
Comment (Abdulazeez-Dolapo) [reply src/components/MeetingTypes/EditMeetingTypeForm.js]: done

**Pull request merged**